app/controllers/services_controller.rb

```diff
@@ -25,6 +25,15 @@ def create
     end
   end
 
+  def destroy
+    service = Service.find_by_id(params[:id])
+    if service.present?
+      DeleteServiceEvent.create(service: service, data: { name: service.name, entity_id: service.entity_id })
+      flash[:success] = t('common.action_successful', name: service.name, action: :deleted)
+    end
```

Comment:
We should have the `else` service not present to be consistent with other destroy methods (i.e. component deletion), but it's a small thing.